test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2612,6 +2643,21 @@ UCS_TEST_P(test_ucp_sockaddr_protocols, am_zcopy_64k,
     test_am_send_recv(64 * UCS_KBYTE);
 }
 
+UCS_TEST_P(test_ucp_sockaddr_protocols, am_rndv_1k, "RNDV_THRESH=0")
+{
+    test_am_send_recv(1 * UCS_KBYTE);
+}
+
+UCS_TEST_P(test_ucp_sockaddr_protocols, am_rndv_64k, "RNDV_THRESH=0")
+{
+    test_am_send_recv(64 * UCS_KBYTE);
+}
+
+UCS_TEST_P(test_ucp_sockaddr_protocols, am_rndv_2mb, "RNDV_THRESH=0")
+{
+    test_am_send_recv(2 * UCS_MBYTE);
+}
```

Comment:
i think no need to test it with different sizes, because rndv protocol is not supposed to change (unlike eager where we have sinle/multi fragments)

Reply:
let's test 2 MBs?

Reply:
let's go with 64K to shorten test exec time?

Reply:
but it will be shortened for Valgrind anyway.
I wanted to make sure that Bcopy/Multi with large number of segments works fine while the connection isn't fully established.

Reply:
imo, it does not matter whether there will be 8 fragments or, say, 20.

Reply:
done